src/v2/Components/Inquiry/Hooks/useArtworkInquiryRequest.ts

```diff
@@ -8,7 +8,12 @@ import {
 type UseArtworkInquiryRequestInput = Omit<
   SubmitInquiryRequestMutationInput,
   "inquireableID" | "inquireableType" | "message"
-> & { relayEnvironment?: Environment; artworkID: string; message: string }
+> & {
+  relayEnvironment?: Environment
+  artworkID: string
+  message: string
+  contactGallery?: boolean | null
```

Comment:
Why was the contactGallery prop needed?

Reply:
> Why was the contactGallery prop needed?

I got pre push type error from prev changes